App.js

```diff
@@ -1,14 +1,17 @@
 const express = require('express'); //require express module
 const linebot = require('linebot');
+const app = express(); //creates an Express application
+const http = require('http');
```

Comment:
can be removed due to express framework already done this for you

Reply:
will fix it.